vendor/github.com/IBM/ibm-csi-common/pkg/utils/constants.go

```diff
@@ -24,7 +24,7 @@ const (
 	// MinimumVolumeSizeInBytes minimum size of the volume in bytes
 	MinimumVolumeSizeInBytes int64 = 10 * GiB
 	// MaximumVolumeSizeInBytes the max allowed capacity
-	MaximumVolumeSizeInBytes int64 = 2 * 1024 * GiB //2000GB = 2TB
+	MaximumVolumeSizeInBytes int64 = 16 * 1024 * GiB //16000GB = 16TB
```

Comment:
This should be taken up automatically by updating the vendor. Instead create PR in csi-common and update the release in go.mod